### ISSUE TITLE
Two reports

### ISSUE THREAD

**Issue by kmbisset89** [Kotlin/kotlinx-kover#351]
Is there a way to run kover twice once generating a report on the entire project and the second time on very specific classes?

**Comment by shanshin** [comment]
Hi,
unfortunately, now there is only one task for generating HTML/XML files.

You may pass an additional parameter when starting the build and change the filters depending on it, e.g.
```
koverReport {
    filters {

        if (project.hasProperty("limitedKoverReport")) {
            includes{
                classes("very.specific.classes.*")
            }
        }

    }
}
```

and pass it during the build
`gradlew koverHtmlReport -PlimitedKoverReport`

**Comment by kmbisset89** [comment]
Does filtering with the word include exclude everything that is not in the include block?

**Comment by shanshin** [comment]
> Does filtering with the word include exclude everything that is not in the include block?

Yes, the `includes` filter excludes all classes from the report except those specified. 
It is acceptable to use `*` and `?` wildcards in class names; `*` means any number of arbitrary characters (including no chars), `?` means one arbitrary character.
